scripts/aws/clodformation_paris_infra.yaml

```diff
@@ -0,0 +1,40 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Deploy PN-delivery and ensure infrastructure
+
+Resources:
+  Keyspace:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Circular Dependencies for resource Keyspace.  Circular dependency with [Keyspace]